.github/workflows/mobile-e2e.yml

```
@@ -116,8 +131,33 @@ jobs:
-          pnpm e2e:build:ios
+          set -euo pipefail
+          echo "Starting iOS app build with timeout..."
+
+          # Add timeout to the build command to prevent hanging
+          if timeout 600 pnpm e2e:build:ios; then
+            echo "✅ iOS build completed successfully"
+          else
+            echo "❌ iOS build timed out or failed"
+            echo "This appears to be a GitHub Actions infrastructure issue"
+            echo "Attempting cleanup and retry..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)